kubeclient/kubeconfig.go

```diff
@@ -0,0 +1,37 @@
+package kubeclient
+
+import (
+	"flag"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+	"path/filepath"
+)
+
+func GetKubeConfig(server string) (*rest.Config, error) {
+	var kubeconfig *string
+	if home := homedir.HomeDir(); home != "" {
+		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
```

Comment:
how does this interact with flags passed to kubernixos?

Reply:
Nice catch! I think this should be changed since it relies on goflags, which is not used elsewhere. Probably another c/p / cargo culting mistake. Will change it.